### ISSUE TITLE
"Copy Stream URL" and "Download" menu options do not honor the selected pull-down option of multiple versions

### ISSUE THREAD

**Issue by gittes** [jellyfin/jellyfin-web#1824]
**Describe The Bug**
When selecting something that has multiple versions in a folder in the library backend (Like "title - [1080P].mkv" and "title - [4K HDR].mkv" in the same title folder) and different version is selected in the web UI it either detects the client source or defaults to the last file listed in the meta data, and doesn't let you download or get the "Copy Stream URL" that it's always locked to by default.

**Steps To Reproduce**
1. Use Safari/Chrome/Firefox using an iPad/iPhone
2. Click on any media menu
3. Click on "Copy Stream URL" or "Download"

**Expected Behavior**
Copy Stream/Download URL (which is the same) should honor what version and options I selected in pull-down menus

**System (please complete the following information):**
 - Platform: macOS
 - Browser: Chrome, Safari
 - Jellyfin Version: 10.6.3

**Additional Context**
Also, could the media info menu option also show the attribute for the MediaSourceId number that is used in URLs? Like in /Items/\<MediaSourceId\>/Download?api_key=#### or in /videos/\<MediaSourceId with hyphens\>/main.m3u8 so it can be at least derived... assuming MediaSoruceId is unique for multiple version of the same item. The edit meta data window also only shows one path, and the media info window shows both/multiple paths info.

Long story short:
How can you specify the desired version to copy/download for an external player when fetching the information from the web UI?
